megalinter/reporters/UpdatedSourcesReporter.py

```diff
@@ -63,6 +64,16 @@ def produce_report(self):
                 f" in folder {updated_sources_dir}.\n"
                 "Download it from artifacts then copy-paste it in your local repo to apply linters updates"
             )
+
+            apply_fixes = config.get_list(self.request_id, "APPLY_FIXES", "none")
+            if apply_fixes:
+                try:
+                    repo = git.Repo(os.path.realpath(self.master.master.github_workspace))
```

Comment:
sorry, master is MegaLinter instance, so it will be self.master.github_workspace :)

Reply:
switched 😄